MIG-1504: MTC 1.8.2 release notes (#67775)
openshift/openshift-docs: modules/migration-mtc-release-notes-1-8-2.adoc

// Module included in the following assemblies:
//
// * migration_toolkit_for_containers/mtc-release-notes.adoc
:_mod-docs-content-type: REFERENCE
[id="migration-mtc-release-notes-1-8-2_{context}"]
= {mtc-full} 1.8.2 release notes

[id="resolved-issues-1-8-2_{context}"]
== Resolved issues

This release has the following major resolved issues:

.Backup phase fails after setting custom CA replication repository

In previous releases of {mtc-full} ({mtc-short}), after editing the replication repository, and adding a custom CA, and the repo is successfully connected, when triggering a migration, there was a failure returned on the backup phase.

.CVE-2023-26136: tough-cookie package before 4.1.3 are vulnerable to Prototype Pollution

In previous releases of ({mtc-short}), versions of the package tough-cookie before 4.1.3, used in {mtc-short}, were vulnerable to Prototype Pollution because of the improper handling of Cookies when using CookieJar in `rejectPublicSuffixes=false` mode.

> It is advised to update to {mtc-short} 1.8.2 or later, which resolve this issue.

Reviewer: ```suggestion
To resolve this issue, update to {mtc-short} 1.8.2 or later .
```
Don't use "advise" or "recommend". We want to be more prescriptive and prevent any user uncertainty. See the SSG entry on "recommend": https://redhat-documentation.github.io/supplementary-style-guide/#recommend

In any case, I think you can simply delete this line. This item is in the "Resolved issues" section, after all.

For more details, see link:https://access.redhat.com/security/cve/cve-2023-26136[(CVE-2023-26136)]

.CVE-2022-25883 openshift-migration-ui-container: nodejs-semver: Regular expression denial of service

> In previous releases of ({mtc-short}), versions of the package semver before 7.5.2, used in {mtc-short}, were vulnerable to Regular Expression Denial of Service (ReDoS) from the function new Range, when untrusted user data is provided as a range.

Reviewer: ```suggestion
In previous releases of ({mtc-short}), versions prior to 7.5.2 of the `semver` package used in {mtc-short} were vulnerable to Regular Expression Denial of Service (ReDoS) attacks when untrusted user data was provided as a range in the new `Range` function.
```
- Put `Range` in backticks because it's a code function
- Rephrase for better flow.

> It is advised to update to {mtc-short} 1.8.2 or later, which resolve this issue.

Reviewer: ```suggestion
To resolve the issue, update to {mtc-short} 1.8.2 or later.
```

Reword, or I think you can simply delete this line. This item is in the "Resolved issues" section, after all.

For more details, see link:https://access.redhat.com/security/cve/cve-2022-25883[(CVE-2022-25883)]

[id="known-issues-1-8-2_{context}"]
== Known issues

> There are no major known issues in this release.

Reviewer: ```suggestion
No major known issues exist in this release.
```
Per the IBM Style guide, restructure sentences to avoid using "there is" and "there are".
